database/sql/04-products.sql

```diff
@@ -149,6 +149,7 @@ INSERT INTO acquirable (id, name, slug) VALUES
     ('5c0f1cfa-bcf8-4587-9513-88cb197ec863', 'ndfd_conus_airtemp', 'ndfd-conus-airtemp'),
     ('d4e67bee-2320-4281-b6ef-a040cdeafeb8', 'hrrr_total_precip','hrrr-total-precip'),
     ('ec926de8-6872-4d2b-b7ce-6002221babcd', 'wrf_columbia_precip','wrf-columbia-precip'),
+    ('552bf762-449f-4983-bbdc-9d89daada260', 'wrf_columbia_t2','wrf-columbia-t2'),
```

Comment:
suggest calling the slug **wrf-columbia-airtemp**

Reply:
Changed